app/forms/application_form.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ApplicationForm
+  include ActiveModel::Naming
+  include ActiveModel::Validations
```

Comment:
Seems like we'll need `include ActiveModel::Conversion` here as well

Reply:
Included.